core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -83,7 +67,7 @@ public static void appendHttpStatusTag(ImmutableList.Builder<Tag> tagListBuilder
         tagListBuilder.add(Tag.of("httpStatus", status.codeAsText()));
     }
 
-    private static void onRequest(RequestLog log, MeterIdPrefixFunction meterIdPrefixFunction, boolean server) {
+    public static void onRequest(RequestLog log, MeterIdPrefixFunction meterIdPrefixFunction, boolean server) {
```

Comment:
Add Javadoc?

Reply:
I thought it was okay because it's in the internal package. but let me revert this anyway